src/ExportPro.Common/ExportPro.Common.Shared/Library/BaseResponseT.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ExportPro.Common.Shared.Library;
+
+public class BaseResponseT<T> : BaseResponse
```

Comment:
I think we can remove T letter at the end of BaseResponse class and I think you can put it in one .cs file